innerprod/simple/ddh_test.go

```diff
@@ -26,14 +26,24 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestSimple_DDH(t *testing.T) {
+type dDHTestParam struct {
+	name string
+	modulusLength int
+	precomputed bool
+}
+
+func testSimple_DDHFromParam(t *testing.T, param dDHTestParam) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [don't use underscores in Go names; func testSimple_DDHFromParam should be testSimpleDDHFromParam](https://app.codacy.com/app/tilenmarc/gofe_2/pullRequest?prid=4432579)